app/app.go

```diff
@@ -366,6 +372,7 @@ type App struct {
 	// simulation manager
 	sm           *module.SimulationManager
 	configurator module.Configurator
+	keepers.AppKeepers
```

Comment:
I believe we will need to not keep two keeper objects one on AppKeepers and the other under app directly.
We can only keep AppKeepers only.

Reply:
same reson
https://github.com/UnUniFi/chain/pull/536#discussion_r1197416143
Since AppKeepers has memory addresses, there will be no impact on performance.

Reply:
Okay